partcad-cli/src/partcad_cli/click/commands/export.py

```
@@ -0,0 +1,125 @@
+    with logging.Process("Export", "this"):
+        ctx.option_create_dirs = create_dirs
+        package = package if package is not None else ""
+        if recursive:
+            start_package = pc_utils.get_child_project_path(ctx.get_current_project_path(), package)
+            all_packages = ctx.get_all_packages(start_package)
+            packages = list(
+                map(
+                    lambda p: p["name"],
+                    list(all_packages),
+                )
+            )
```

Ruff (0.8.2)

Unnecessary `map()` usage (rewrite using a list comprehension)

Replace `map()` with a list comprehension

(C417)